gen/schema/schema.yaml

```diff
@@ -25,6 +25,7 @@ no_data_source: bool(required=False) # Set to true if no data source should be c
 no_resource: bool(required=False) # Set to true if no resource should be created
 get_before_delete: bool(required=False) # This does a GET "all" before doing a DELETE, which is a workaround for an API issue with some endpoints where the reference count is not updated otherwise
 delete_mutex: bool(required=False) # Set to true if DELETE operation is mutually exclusive with other DELETE operations
+exclude_minimum_test: bool(required=False) # Set to true to prevent "minimum" resource acceptance testing
```

Comment:
We already have this in other implementations, though it is named `skip_minimum_test` there (https://github.com/CiscoDevNet/terraform-provider-catalystcenter/blob/95eee85ea31fd2788a5fc8e540df994a7a9aefcd/gen/schema/schema.yaml#L38C1-L38C18). Would it be ok to rename it accordingly?